File: Section Reports/Layout/SubReport/VB.NET/SubReport/rptParamSub.vb
Public Class rptParamSub
	Inherits GrapeCity.ActiveReports.SectionReport
	Public Sub New()
		MyBase.New()
		InitializeComponent()
		Dim dS As New Data.OleDBDataSource()
		dS.ConnectionString = My.Resources.ConnectionString
		dS.SQL = "SELECT * FROM products INNER JOIN categories ON products.categoryid = categories.categoryid WHERE products.supplierID = <%SupplierID%>"
		DataSource = dS
	End Sub
	Private Sub rptChild_ReportStart(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.ReportStart
		ShowParameterUI = False
	End Sub
End Class

File: Section Reports/Layout/SubReport/C#/SubReport/rptParamSub.cs
using System;
namespace GrapeCity.ActiveReports.Samples.SubReport
{
	/// <summary>
	/// Description of the outline of the  rptParamSub.
	/// 
	/// </summary>
	public partial class rptParamSub : GrapeCity.ActiveReports.SectionReport
	{
		public rptParamSub()
		{
			//
			// ActiveReport designer support is required.
			//
			//
			InitializeComponent();
			Data.OleDBDataSource dS = new Data.OleDBDataSource();
			dS.ConnectionString = Properties.Resources.ConnectionString;			
			dS.SQL = "SELECT * FROM products INNER JOIN categories ON products.categoryid = categories.categoryid WHERE products.supplierID = <%SupplierID%>";
			DataSource = dS;
		}
		private void rptParamSub_ReportStart(object sender, EventArgs e)
		{
			ShowParameterUI = false;
		}
	}
}
